src/generators/markdowndocgenerator/itemrenderer.tsx

```diff
@@ -21,7 +21,7 @@ export function markdownDocItemRenderer(
   };
   let fontSize = '9px';
   let numbering = item.numbering && options.numbering ? item.numbering : '';
-  if (item.type === 'header') {
+  if (item.type === INotebookHeadingTypes.header) {
```

Comment:
Is this check necessary? It doesn't seem like the markdown doc should depend on the notebook heading types. After all, aren't they all `header`?

Reply:
Yes they are. Thank you for the catch!